pdf_report/transform_pdf.py

```diff
@@ -0,0 +1,76 @@
+"""Create PDF with train summary details"""
+
+from psycopg2.extensions import connection
+
+from extract_pdf import get_connection, query_db
+
+# [RealDictRow({'station_id': 7, 'waypoint_id': 8407, 'run_date': datetime.date(2024, 7, 22), 'booked_arrival': datetime.datetime(2024, 7, 22, 23, 35), 'actual_arrival': datetime.datetime(2024, 7, 22, 23, 34), 'booked_departure': datetime.datetime(2024, 7, 22, 23, 39), 'actual_departure': datetime.datetime(2024, 7, 22, 0, 1), 'service_id': 5097, 'station_crs': 'YRK', 'station_name': 'York Rail Station'})]
```

Comment:
Was this comment a personal note? perhaps this note is no longer needed.
```suggestion
```

Reply:
ew why are there comments in a pull request? fix up